website/server/extractDocs.js

```diff
@@ -206,30 +208,140 @@ function componentsToMarkdown(type, json, filepath, idx, styles) {
 
 let componentCount;
 
+function getTypedef(filepath, fileContents, json) {
+  let typedefDocgen;
+  try {
+    typedefDocgen = docgen.parse(
+      fileContents,
+      docgenHelpers.findExportedType,
+      [docgenHelpers.typedefHandler]
+    ).map((type) => type.typedef);
+  } catch (e) {
+    // Ignore errors due to missing exported type definitions
+    if (e.message.indexOf(docgen.ERROR_MISSING_DEFINITION) !== -1) {
+      console.error('Cannot parse file', filepath, e);
+    }
```

Comment:
Ok, but next line the code proceeds, you probably need to add `throw e` after the `if` clause